src/main/java/dev/mccue/resolve/maven/MavenCentralDownloader.java

```diff
@@ -0,0 +1,92 @@
+package dev.mccue.resolve.maven;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.channels.Channels;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Creates an object from a specified path at maven central. Allows for the downloading of POMS and Jars.
+ */
+public class MavenCentralDownloader {
+
+    private final String mavenBaseURL = "https://repo.maven.apache.org/maven2/";
```

Comment:
Remember that this is configurable - there are multiple mirror urls even for just maven central

Reply:
As discussed on Thursday, we will keep it as just maven for now and will add/change in the future